content/chainguard/chainguard-images/reference/postgres/getting-started-postgres/index.md

```diff
@@ -0,0 +1,194 @@
+
+```sh
+git clone --sparse https://github.com/chainguard-dev/edu-images-demos.git
+```
+
+Because this guide's demo application code is stored in a repository with other examples, we don't need to pull down every file from this repository. For this reason, this command includes the `--sparse` option. This will initialize a sparse-checkout file; this causes the working directory to contain only the files in the root of the repository until the sparse-checkout configuration is modified.
+
+Navigate into this new directory.
+
+```sh
+cd edu-images-demos
+```
+
+To retrieve the files you need for this tutorial's sample application, run the following `git` command.
+
+```sh
+git sparse-checkout set postgres
+```
+
+This modifies the sparse-checkout configuration initialized in the previous `git clone` command so that the checkout only consists of the repo's `postgres` directory.
+
+Navigate into this new directory.
+
+```sh
+cd postgres/
+```
+
+From here, you can run the application and use a web browser to observe it working in real time, which we'll do in the next section.
+
+
+## Step 2: Inspect, run, and test the sample application
+
+We encourage you to check out [the application code on GitHub](https://github.com/chainguard-dev/edu-images-demos/tree/main/postgres) to better understand how this application works, but we'll provide a brief overview here.
+
+This demo creates a LEPP (Linux, (E)Nginx, PostgreSQL and PHP-FPM) environment based on Wolfi Chainguard Images. We will use Docker Compose to bring up the environment, which will spin up three containers: an `app` container, a `postgres` container, and an `nginx` container. These will run as services.
+
+Once the environment is up, you can visit the demo in your web browser. The `index.php` file contains code that does the following:
+
+* Connects to the PostgreSQL server running in the `postgres` container
+* Creates a new table named `data` if it doesn't already exist
+* Inserts a new entry into the table with a random number
+* Queries the table to show all the entries
+
+Every time you reload the page, a new entry will be added to the table.
+
+Note that this application includes a Dockerfile. 
+
+```sh
+cat Dockerfile
+```
+```
+FROM cgr.dev/chainguard/php:latest-fpm-dev
+
+USER root
+RUN apk update && apk add php-pgsql
+
+USER php
+```
+
+This Dockerfile takes the public `php:latest-fpm-dev` Chainguard Image and installs the `php-pgsql` package onto it. This image comes with drivers that allow PHP applications to connect to MySQL or MariaDB databases by default but it doesn't have an equivalent for PostgreSQL. For this reason, we use this Dockerfile to install this package in order for the PHP application to be able to connect to the Postgres database. 
+
+Execute the following command to build an image with this Dockerfile, and then create and start each of the three containers and bring up the application.
+
+```sh
+docker compose up -d
```

Comment:
On macOS I always need to add `sudo`, so I would just add a line saying if there is an error to run `sudo` with it